torch/csrc/jit/tensorexpr/kernel.h

```diff
@@ -274,9 +292,12 @@ class TORCH_API TensorExprKernel {
   std::vector<ExprHandle> getInputStrides(
       const torch::jit::Value* input,
       const std::vector<ExprHandle>& inputTensorDims);
-  std::vector<torch::jit::StrideInput>& getSymbolicInputStrideDesc(
+  std::vector<torch::jit::StrideInput>& getSymbolicStrideDesc(
       const torch::jit::Value* value);
 
+  // Abstract graph optimizations as a single function.
```

Comment:
Nit: the comment would be more helpful if it explains why/when we need this, what we do there (just on a high level).